docs/EKS-cni-support.md

```diff
@@ -0,0 +1,22 @@
+# CNI monitoring special configuration updates for EKS
+
+AWS EKS uses [CNI](https://github.com/aws/amazon-vpc-cni-k8s) networking plugin for pod networking in Kubernetes using Elastic Network Interfaces on AWS
+
+One fatal issue that can occur is that you run out of IP addresses in your eks cluster. (Generally happens due to error configs where pods keep scheduling).
+
+You can monitor the `awscni` using kube-promethus with : 
+```
```

Comment:
We manage all examples in the examples directory and then embed them in docs via `embedmd`. I would suggest looking at some other docs to see an example of that. As a bonus all snippets in the examples directory get run on CI so we know that they in fact work :)

Reply:
tried with `embedmd` b03ff4f5931c82c4acaf49a43972d3cfdf94f01f edb327531d88d876b6b3532f2f759aefb1625280

Reply:
fixed the bug with example..
3c4dbc52d9b7727439d1d8595791c351b1c391fc d4ba158f9baab0a89466f6405c2d937954582247 resolves the issues.